demos/mri_reconstruction_demo/cpp/main.cpp

```diff
@@ -38,16 +38,31 @@ int main(int argc, char** argv) {
         return 0;
     }
 
-    slog::info << *InferenceEngine::GetInferenceEngineVersion() << slog::endl;
-    InferenceEngine::Core ie;
+    slog::info << ov::get_openvino_version() << slog::endl;
+    ov::runtime::Core core;
 
-    InferenceEngine::CNNNetwork net = ie.ReadNetwork(FLAGS_m);
-    net.getInputsInfo().begin()->second->setLayout(InferenceEngine::Layout::NHWC);
+    slog::info << "Reading model " << FLAGS_m << slog::endl;
+    std::shared_ptr<ov::Model> model = core.read_model(FLAGS_m);
+    logBasicModelInfo(model);
 
-    InferenceEngine::ExecutableNetwork execNet = ie.LoadNetwork(net, FLAGS_d);
-    logExecNetworkInfo(execNet, FLAGS_m, FLAGS_d);
+    std::string outputTensorName = "";
+    for (const auto& output : model->outputs()) {
+        if (output.get_any_name().find("89") != std::string::npos) {
+            outputTensorName = output.get_any_name();
+        }
+    }
+    if (outputTensorName == "") {
+        throw std::logic_error("Not found suitable output!");
+    }
```

Comment:
The demo expects the model with 1 output tensor, I think in that case we might not check the output name. The check is preferable for models with 2 or more outputs to extract data from specific tensor. So you can simplify it like this:
```suggestion
   OPENVINO_ASSERT(model->outputs().size() == 1, "The demo supports models with 1 output only");
   std::string outputTensorName = model->outputs()[0].get_any_name();
```

Reply:
I see that model has 2 outputs:
```
[ INFO ] Reading model ../../intel64/Debug/hybrid-cs-model-mri.xml
[ INFO ] model name: hybrid-cs-model-mri
[ INFO ] inputs:
[ INFO ] Func/StatefulPartitionedCall/input/_0:0, f32, {1,256,256,2}
[ INFO ] outputs:
[ INFO ] Func/StatefulPartitionedCall/output/_90:0, f32, {1,256,256,2}
[ INFO ] Func/StatefulPartitionedCall/output/_89:0, f32, {1,256,256,1}
```
And we need the one with `89` in name.

Reply:
Maybe second output emerged after recent changed that were done in MO.

Reply:
Yeah, the number of outputs is probably changed with IR version 11. Then I will add the same check to the Python demo. Thank you for the clarification!